README.md

```diff
@@ -55,7 +55,7 @@ and also follows [Kaldi](http://kaldi-asr.org/) style data processing, feature e
 - **Transducer** based end-to-end ASR
   - Architecture:
     - RNN-based encoder and decoder.
-    - Custom encoder and decoder supporting Transformer, Conformer (encoder), TDNN (encoder) and causal Conv1D (decoder) blocks.
+    - Custom encoder and decoder supporting Transformer, Conformer (encoder), 1D Conv (encoder) and causal 1D Conv (decoder) blocks.
```

Comment:
TDNN is more familiar in the speech community.
So, how about keeping TDNN in some ways? (e.g., 1D Conv (TDNN encoder